3rdparty/google_cloud_texttospeech/bin/text2wave

```diff
@@ -29,17 +100,37 @@ if __name__ == '__main__':
     if args.evaluate in ['(ja)', '(ja-JP)']:
         language_code = 'ja-JP'
         name = 'ja-JP-Wavenet-A'
-    voice = texttospeech.types.VoiceSelectionParams(
+
+    if cache_enabled:
+        cache_dir = get_cache_dir()
+        md5 = checksum_md5(args.text)
+        cache_filename = os.path.join(
+            cache_dir,
+            '--'.join([md5, language_code, name, str(speaking_rate)])
+            + '.mp3')
+        if os.path.exists(cache_filename):
+            shutil.copy(cache_filename, args.output)
```

Comment:
how about `rospy.loginfo` something like "Using caching sound file ('/home/..../....wav') for 'Hello'"

Reply:
In current implementation, `text2wave` is not intended to be used in ROS because not called `rospy.init_node('nodename')` at first.
I agree with your idea of adding logging.
Is it okay to add `rospy.init_node` at the beginning of text2wave's main function?

Reply:
Oh, we need to use `print` here...

Reply:
I add `print` for logging.
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/267/files#diff-eae6b228234bee5fba1f7f9e62919bf189b3b6a545dbe0f2d0a33c88a2f20962R115